std/json.d

```diff
@@ -346,10 +346,11 @@ struct JSONValue
             type_tag = JSON_TYPE.STRING;
             store.str = arg;
         }
-        else static if(is(T : char[]))
+        else static if(isNarrowString!T)
```

Comment:
You need to use [`isSomeString`](http://dlang.org/phobos/std_traits#isSomeString), because [`isNarrowString`](http://dlang.org/phobos/std_traits#isNarrowString) would not allow `dchar`-based strings